src/devtools/Types.ts

```diff
@@ -17,18 +40,6 @@
  * @property {object} graph
  */
 
-/**
```

Comment:
Wouldn't this break the JSDoc export? Don't we need to keep the annotation with the change above?

Reply:
Good point. I was unaware that JSDoc is actively used in this project and indeed, [JSDoc is incompatible with TypeScript](https://stackoverflow.com/questions/51720894/is-there-any-way-to-use-jsdoc-with-ts-files-maybe-transpile-with-babel-the). 

Can you explain the purpose/intentions of JSDoc here? The docs seems to only be executed as side-effect of tests and are not hosted anywhere.